pittapi/course.py

```diff
@@ -215,7 +215,16 @@ def get_course_details(term: Union[str, int], subject: str, course: Union[str, i
         )
 
     return CourseDetails(
+<<<<<<< HEAD
+        course=Course(
+            subject_code=subject,
+            course_number=course,
+            course_id=internal_course_id,
+            course_title=course_title,
+        ),
+=======
         course=Course(subject_code=subject, course_number=course, course_id=internal_course_id, course_title=course_title),
+>>>>>>> 5707e24e2f881755bb429b3b1a0f2245940b46d6
```

Comment:
Some of the files have merge conflicts that weren't properly resolved and need to be cleaned up, e.g.
```suggestion
        course=Course(
            subject_code=subject,
            course_number=course,
            course_id=internal_course_id,
            course_title=course_title,
        ),
```
The stuff between the `<<<` and `===` is code from upstream and the stuff between the `===` and `>>>` is code from your local git repo, and this is a merge conflict because the two versions differ too much for git to automatically merge your changes in. Please look in the other modified files for anything like this and resolve those conflicts (i.e., choose one version or the other and clean up any labels that git added).